src/sidebar/components/MarkdownEditor.tsx

```diff
@@ -257,13 +257,8 @@ function TextArea({
 
       const term = termBeforePosition(textarea.value, textarea.selectionStart);
       const isAtMention = term.startsWith('@');
-      setPopoverOpen(isAtMention);
+      setSuggestionsPopoverOpen(isAtMention);
       setActiveMention(isAtMention ? term.substring(1) : undefined);
-
-      // Reset highlighted suggestion when closing the popover
-      if (!isAtMention) {
-        setHighlightedSuggestion(0);
-      }
```

Comment:
This whole callback is now called unconditionally `onInput`, where we also set `setHighlightedSuggestion(0);`, so this check is no longer needed.